spec/document_cloud_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'spec_helper'
+
+describe DocumentCloud do
+  it 'has a version number' do
+    expect(DocumentCloud::Version).not_to be nil
+  end
+
+  it 'escapes characters in email addresses and passwords' do
+    email = 'test@example.com'
+    password = 'abc123$@#^&*%'
+
+    DocumentCloud.configure do |config|
+      config.email = email
+      config.password = password
+    end
+
+    expect(DocumentCloud.client.instance_variable_get(:@email)).to eq CGI.escape(email)
```

Comment:
This isn't necessarily the nicest way to test this. Reaching in with `instance_variable_get` is something of an antipattern, and I'll try other approaches.

Reply:
It's just a test, so if it works and handles the case the test is for, im good with it :)